en/docs/learn/analytics/customizing-analytics-dashboards.md

```diff
@@ -0,0 +1,27 @@
+#Customizing Analytics Dashboards
+
+API Manager analytics provides three types of dashboards which are APIM Admin, APIM Developer Portal and APIM Publisher.
+
+The users will not be allowed to modify the default dashboards, i.e either to modify the layout of the widget or add custom widgets to a particular dashboard. If it is required to modify the default dashboard, a copy of the dashboard should be created and the modification should be applied to that copy as described below.
```

Comment:
"If it is required to modify one of the default dashboards"

Reply:
Resolved with e0624b9b657a46ee271b237d2dcc9e7fb652a7b1